### PR TITLE
Cypress

### PR DESCRIPTION
**Pull request by eugene-chr**

### CODE REVIEW

File: cypress/integration/projects.spec.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  const nanoid = customAlphabet('1234567890abcdefghi', 10);
+  const email = `${nanoid()}@test.com`;
+  const user = {
+    email,
+    password: 'z1x2c3v4',
+  };
+
+  before(() => {
+    cy.firebaseLogout().then(() => {
+      cy.firebaseCreateUser(user).then(() => {
```
Comment (plzen) [inline cypress/integration/projects.spec.js:14]: Looks like you can use chain linear:

```
cy.firebaseLogout()
  .then(() => cy.firebaseCreateUser(user))
  .then(() => cy.visit('/projects'))
```

File: cypress/integration/projects.spec.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      cy.dataCy('project-name').type(`${projectName}{enter}`);
+      cy.dataCy('field-error-project-name')
+        .should('exist')
+        .and('include.text', 'The project with such name does already exist.');
+    });
+    cy.dataCy('project-form').buttonDisabled('button-submit');
+    cy.dataCy('button-cancel').click();
+  });
+
+  it.skip('should show manage buttons when hover over an project', () => {
```
Comment (plzen) [inline cypress/integration/projects.spec.js:63]: Can't you use `.click({ force: true })` here?

File: cypress/integration/projects.spec.js (hunk 1 above)
Comment (eugene-chr) [reply cypress/integration/projects.spec.js:14]: done

File: cypress/integration/projects.spec.js (hunk 2 above)
Comment (eugene-chr) [reply cypress/integration/projects.spec.js:63]: here I will live it for future features from cypress to `hover` over elements. I clicked buttons without hovering.

**Pull request merged**